### PR TITLE
Rebalances stunprods

### PR DESCRIPTION
**Pull request by S34NW**
## What Does This PR Do
Increases stam damage to be inline with batons
Makes it bulky
Makes it fit in belt slot

https://www.paradisestation.org/forum/topic/22801-stunprod-rework/#comment-166924

## Why It's Good For The Game
They are weak as hell currently.
This will maybe fix it? I am unsure, but it is a step in the right direction.

## Testing
~~I didn't, since its a two-line change.~~

Nocell, Cell, Active
image

## Changelog
:cl:
tweak: Rebalances stunprods to do the same damage as batons but is bulky
/:cl:

### CODE REVIEW

**Comment by roarbark** [comment]
do stunprods get an oncharacter sprite when in the belt slot? also- armours dont fit these in their suit storage- maybe they should?

**Comment by Inadvizable** [comment]
> do stunprods get an oncharacter sprite when in the belt slot? also- armours dont fit these in their suit storage- maybe they should?

Yeah - there should be a sprite, I am not sure if there is and I cannot check right now, however. As for fitting on armor - maybe? I don't think it's really needed. Could be a bit too convenient.

**Comment by GDNgit** [comment]
shawn you just removed the stamina damage from it

**Comment by S34NW** [comment]
> shawn you just removed the stamina damage from it

no

File: code/game/objects/items/weapons/stunbaton.dm [hunk 1]
```diff
@@ -243,9 +243,9 @@
 	force = 3
 	throwforce = 5
 	knockdown_duration = 6 SECONDS
-	stam_damage = 40 // three hits to crit
```
Comment (Inadvizable) [inline code/game/objects/items/weapons/stunbaton.dm:246]: Perhaps you should have tested it after all,

**Comment by S34NW** [comment]
> IT'S ALSO A WEBEDIT

it is not

File: code/game/objects/items/weapons/stunbaton.dm (hunk 1 above)
Comment (S34NW) [reply code/game/objects/items/weapons/stunbaton.dm:246]: no this is now how it works

Comment (Inadvizable) [reply code/game/objects/items/weapons/stunbaton.dm:246]: OH WAIT ITS NOT THE PARENT OF BATON LMAO IM DUMB

Comment (hal9000PR) [reply code/game/objects/items/weapons/stunbaton.dm:246]: removing the stam damage, means that it inherits the stamina damage from the stun baton, EG 60

**Comment by S34NW** [comment]
On-character sprite added, it's horrific but a stopgap. Save me spriters

**Comment by Inadvizable** [comment]
> On-character sprite added, it's horrific but a stopgap. Save me spriters

add: Soul

**Comment by Coolrune206** [comment]
I don't dislike this in concept, but I feel it runs into the same issue most items face when they're balanced by being made bulky - Science and their Bags of Powergaming. Is there any way to maintain the stunprod's malus of storage woes in face of Science's mass-printable BoHs?

**Comment by S34NW** [comment]
> I don't dislike this in concept, but I feel it runs into the same issue most items face when they're balanced by being made bulky - Science and their Bags of Powergaming. Is there any way to maintain the stunprod's malus of storage woes in face of Science's mass-printable BoHs?

Not especially, that requires changes to BoH

**Comment by Inadvizable** [comment]
> I don't dislike this in concept, but I feel it runs into the same issue most items face when they're balanced by being made bulky - Science and their Bags of Powergaming. Is there any way to maintain the stunprod's malus of storage woes in face of Science's mass-printable BoHs?

That's a BoH problem. That item keeps causing issues and I'm personally quite tired of it having influence on so many other items' balance.

**Review by hal9000PR — APPROVED** [review]
peak codersprite energy

**Pull request merged**